unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2071,21 +2071,21 @@ runScheme file args = do
   unless success $
     Cli.returnEarly (PrintMessage "Scheme evaluation failed.")
 
-buildScheme :: String -> String -> Cli ()
-buildScheme main file = do
-  ensureSchemeExists
-  statDir <- getSchemeStaticLibDir
-  genDir <- getSchemeGenLibDir
-  buildRacket genDir statDir main file
-
-buildRacket :: String -> String -> String -> String -> Cli ()
-buildRacket genDir statDir main file =
-  let args = ["-l", "raco", "--", "exe", "-o", main, file]
-      opts = racketOpts genDir statDir args
-   in void . liftIO $
-        catch
-          (True <$ callProcess "racket" opts)
-          (\(_ :: IOException) -> pure False)
+-- buildScheme :: String -> String -> Cli ()
+-- buildScheme main file = do
+--   ensureSchemeExists
+--   statDir <- getSchemeStaticLibDir
+--   genDir <- getSchemeGenLibDir
+--   buildRacket genDir statDir main file
+
+-- buildRacket :: String -> String -> String -> String -> Cli ()
+-- buildRacket genDir statDir main file =
+--   let args = ["-l", "raco", "--", "exe", "-o", main, file]
+--       opts = racketOpts genDir statDir args
+--    in void . liftIO $
+--         catch
+--           (True <$ callProcess "racket" opts)
+--           (\(_ :: IOException) -> pure False)
```

Comment:
Keep these in for now for reference? Or accidental inclusion?

Reply:
Just haven't deleted them yet.